validator/testcases/installrdf.py

```diff
@@ -93,6 +94,21 @@ def _test_rdf(err, install):
         elif predicate == 'multiprocessCompatible':
             err.save_resource('is_multiprocess_compatible', value == 'true')
 
+        elif predicate == 'hasEmbeddedWebExtension':
+            err.notice(('testcases_installrdf',
+                        '_test_rdf',
+                        'hasEmbeddedWebExtension'),
+                       'This add-on is an hybrid add-on',
```

Comment:
Actually, would you mind changing this to `This add-on contains an embedded webextension.`, please? I realize it makes more sense to stay consistent with our terminology and we use "embedded webextensions" on MDN.